pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "trefoil"
+version = "0.1.0"
+description = "Geospatial operations with NetCDF files and numpy arrays."
+authors = ["Conservation Biology Institute"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
We need to add `six` to the dependencies. I tried importing a trefoil module and got an error about no module `six`.

Reply:
Added